.travis.yml

```diff
@@ -54,7 +54,7 @@ before_install:
   - pip install typing
   - |
     if [[ $TRAVIS_PYTHON_VERSION == 3.6 ]]; then
-      pip install onnxruntime
+      pip install -q --user -i https://test.pypi.org/simple/ ort-nightly==1.0.0.dev1123
```

Comment:
Do you have an idea when the new stable version of the `onnxruntime` will be available?

Reply:
not sure, I will ask and let you know